hash_practice/exercises.py

```diff
@@ -1,19 +1,52 @@
 
+def create_anagram_key(string):
+    return ''.join(sorted(string))
+
+
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n*m)
+        Space Complexity: O(n)
     """
-    pass
+    grouped_words = {}
+    anagrams = []
+
+    for ele in strings:
+        if grouped_words.get(create_anagram_key(ele)) == None:
+            grouped_words[create_anagram_key(ele)] = [ele]
+        else:
+            grouped_words[create_anagram_key(ele)].append(ele)

```

Comment:
Instead of calling your helper function three times, consider calling it just once, at the top of the for loop body.
```suggestion
        anagram_key = create_anagram_key(ele)
        if grouped_words.get(anagram_key) == None:
            grouped_words[anagram_key] = [ele]
        else:
            grouped_words[anagram_key].append(ele)

```